addons/website_blog/static/src/scss/website_blog.scss

```diff
@@ -151,14 +151,6 @@ $o-wblog-loader-size: 50px;
         // "Regular Cover" design sizes
         // ==============================================
         &.o_wblog_post_page_cover_regular {
-            &.o_full_screen_height {
-                min-height: 350px;
-            }
-
-            &.o_half_screen_height {
-                min-height: 250px;
-            }
```

Comment:
Discussed with JKE, we think we should fix those rules instead of removing them :wink: And in another commit as this is not related to what you are fixing, you found another bug ^^ introduced by mistake with https://github.com/odoo/odoo/commit/7cc6fdf36569be85e995c962321f2926822ac604

13.0:
![Screenshot from 2021-06-10 17-13-14](https://user-images.githubusercontent.com/10338094/121551771-3573c880-ca10-11eb-85ef-66f4fb4d78c0.png)

14.0:
![Screenshot from 2021-06-10 17-13-21](https://user-images.githubusercontent.com/10338094/121551805-3efd3080-ca10-11eb-8e20-d95b19e595a7.png)

For someone who started with 14.0, it would just thought that was ugly (and hopefully would prefer the initial version) and for someone coming from 13.0, the fix is "important" :wink:

Reply:
Hehe, ok, I will reorganize the fixes